### DISCUSSION TITLE
What happened to kml_output.xml?

### DISCUSSION THREAD

**Discussion by jonsberndt** [JSBSim-Team/jsbsim#705] [Q&A]
At one time there was a kml_output.xml file in the data_output/ directory. Looks like that is no longer there. I was wondering what happened to that, and if it has been superseded in one way or another?

**Comment by jonsberndt** [comment]
> At one time there was a kml_output.xml file in the data_output/ directory. Looks like that is no longer there. I was wondering what happened to that, and if it has been superseded in one way or another?

Actually, looks like that file is in the scripts/ directory. I can't recall why I have it in my data_output/ directory, too.

**Reply by bcoconni** [reply]
AFAICT the file `kml_output.xml` has been always been located in `scripts` as [the commit history shows](https://github.com/JSBSim-Team/jsbsim/commits/master/scripts/kml_output.xml). It has remained unmodified for 11 years.

**Reply by jonsberndt** [reply]
Looks like I copied it over to the data_output/ directory in my own working directory for some reason. So ... no problem!
